src/scr_copy.c

```diff
@@ -265,6 +266,7 @@ static int scr_bool_have_files(scr_filemap* map)
 
   return have_files;
 }
+#endif
```

Comment:
why comment it out but keep it? Is it expected to be needed again in the future?

Reply:
Fixed in latest push